scripts/test_parse_directory.py

```diff
@@ -182,10 +182,15 @@ def parse_directory(
                 if not short:
                     report_status(succeeded=True, file=file, verbose=verbose)
             except Exception as error:
-                report_status(
-                    succeeded=False, file=file, verbose=verbose, error=error, short=short
-                )
-                errors += 1
+                try:
+                    ast.parse(file)
+                except Exception:
+                    pass
```

Comment:
Nit: Can we log something here so we know that those files fail also in Python 3?